x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/overlay.tsx

```diff
@@ -119,7 +119,7 @@ export const NodeContextPopover = ({
                       >
                         <FormattedMessage
                           id="xpack.infra.infra.nodeDetails.createAlertLink"
```

Comment:
Is the repetition `infra.infra` in this text `xpack.infra.infra.nodeDetails.createAlertLink` intended? In the waffle component later on you use `xpack.infra.nodeContextMenu.createAlertLink`. It would be good to be consistent. How about using `xpack.infra.nodeDetails.createAlertLink` instead?

Reply:
I can't speak to this from a development standpoint since I was just updating the text here. Would it be alright to resolve this to get it in before FF and then I will create another issue that addresses this and @neptunian's `https://github.com/elastic/kibana/pull/103589/files/9ba3ce9ddf27abf120c630c9b32e5831ec90392c#r660844635` below?

Reply:
Yep sure makes total sense